### ISSUE TITLE
HTTPS

### ISSUE THREAD

**Issue by bromagosa** [bromagosa/beetleCloud#6]
Everything should travel over HTTPS, or at least login credentials!

**Comment by b1r63r** [comment]
I would guess this bug keeps quite a few visitors from creating a user. Especially now that browsers complain so loudly.

**Comment by bromagosa** [comment]
Does your browser complain when you try to sign up / login? I don't get any warnings at all.

This is still important, obviously :)

**Comment by b1r63r** [comment]
Yes, it definitely complains very visibly.3. apr. 2017 12.32 p.m. skrev Bernat Romagosa <notifications@github.com>:Does your browser complain when you try to sign up / login? I don't get any warnings at all.
This is still important, obviously :)

**Comment by sciencemonk** [comment]
Second request HTTPS! Any updates on this?

(Huge fan of Beetleblocks btw, we run a micro-school and our students love it)

**Comment by R4356th** [comment]
> Does your browser complain when you try to sign up / login? I don't get any warnings at all.
> 
> This is still important, obviously :)

It does not for me, @bromagosa .
